### PR TITLE
PDP-1185: Second batch of reworks

### PR DESCRIPTION
**Pull request by danibishop**

### CODE REVIEW

File: lib/count_loan_declined_events/main.py [hunk 1]
```diff
@@ -63,7 +64,7 @@ async def get_count_loan_declined_events(
     token = os.environ["PNGME_TOKEN"]
     client = AsyncClient(token)
 
-    utc_endtime = datetime(2021, 10, 1)
+    utc_endtime = datetime(2021, 11, 1)
```
Comment (danibishop) [inline lib/count_loan_declined_events/main.py:67]: This was returning zero, so I picked a new date to get some value.

File: lib/count_loan_repayment_events/main.py [hunk 2]
```diff
@@ -24,31 +24,33 @@ async def get_count_loan_repayment_events(
...
-    for alert in record_list:
-        if alert["ts"] >= utc_starttime.timestamp() and alert["ts"] < utc_endtime.timestamp():
-            count_loan_repayment_events += 1
+    # STEP 4: count number of alerts
+    return len(all_alerts)
 
-    return count_loan_repayment_events
 
 
 if __name__ == "__main__":
```
Comment (JaniceCanedo) [inline lib/count_loan_repayment_events/main.py:56]: Did you need to change the date on this?

File: lib/count_overdraft_events/main.py [hunk 3]
```diff
@@ -24,31 +24,32 @@ async def get_count_overdraft_events(
...
-    for alert in record_list:
-        if alert["ts"] >= utc_starttime.timestamp() and alert["ts"] < utc_endtime.timestamp():
-            count_overdraft_events += 1
-    
-    return count_overdraft_events
+    # STEP 4: count number of alerts
+    return len(all_alerts)
 
 
 if __name__ == "__main__":
```
Comment (JaniceCanedo) [inline lib/count_overdraft_events/main.py:55]: Did you need to change the date on this one like the other two?

File: lib/count_loan_repayment_events/main.py (hunk 2 above)
Comment (danibishop) [reply lib/count_loan_repayment_events/main.py:56]: I only changed the date in features where originally I had not nice numbers in the first refactor while testing. In this refactor I checked that we had relevant numbers

**Review by JaniceCanedo — APPROVED** [review]

**Pull request merged**